WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersViewModel.kt

```diff
@@ -237,6 +237,10 @@ class BloggingRemindersViewModel @Inject constructor(
         onSettingsItemClicked(siteStore.getLocalIdForRemoteSiteId(remoteSiteId), NOTIFICATION_SETTINGS)
     }
 
+    fun onBloggingPromptSchedulingRequested(siteId: Int) {
+        showBottomSheet(siteId, PROLOGUE, PUBLISH_FLOW)
```

Comment:
Sorry, I missed it before, but we should probably add a separate source here instead of `PUBLISH_FLOW`, something like `BLOGGING_PROMPT_ONBOARDING` :)

Reply:
Good point, thanks! 👍 
Fixed